### PR TITLE
5 move to logger instead of prints

### LINKED ISSUE #5: Move to LOGGER instead of prints
Lets switch codebase info and debug printing to LOGGER for better log management.

- Differentiate logs to INFO, WARNING, DEBUG, ERROR levels.
- It should stay at INFO level for users, developers can manually switch to DEBUG version when needed.
- No print function should left.

### PR DESCRIPTION
**Pull request by AimiPia**
Logger Added

### CODE REVIEW

File: utils/logger.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    interval=1,
+    backupCount=10,
+    encoding="utf-8"
+)
+file_handler.setLevel(logging.DEBUG)
+file_format = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+file_handler.setFormatter(file_format)
+
+console_handler = logging.StreamHandler()
+console_handler.setLevel(logging.DEBUG)
```
Comment (KBurakTokmak) [inline utils/logger.py]: lets set console logger to INFO, for users. Devs can switch to debug manually or look at the file logger.

File: utils/logger.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+console_handler.setFormatter(file_format)
+
+logger = logging.getLogger()
+logger.setLevel(logging.DEBUG)
```
Comment (KBurakTokmak) [inline utils/logger.py]: lets set console logger to INFO, for users. Devs can switch to debug manually or look at the file logger.

**Review by KBurakTokmak — CHANGES_REQUESTED** [review]
2 smol changes

File: utils/logger.py (hunk 2 above)
Comment (AimiPia) [reply utils/logger.py]: Done

File: utils/logger.py (hunk 1 above)
Comment (AimiPia) [reply utils/logger.py]: Done

**Review by KBurakTokmak — APPROVED** [review]
LGTM

**Pull request merged**